src/Config.php

```
@@ -164,4 +172,49 @@ private function getRouterMap(InjectorInterface $injector): ?Map
+    private function getQueryClasses(InjectorInterface $injector): array
+    {
+        // Try to get Ray\MediaQuery\Queries if available
+        $queriesClass = 'Ray\\MediaQuery\\Queries';
+        if (! class_exists($queriesClass)) {
+            return [];
+        }
+
+        try {
+            /** @var class-string $queriesClass */
+            /** @var object{classes: list<class-string>} $queries */
+            $queries = $injector->getInstance($queriesClass);
```

PHPStan (2.1.33)

PHPDoc tag @var with type class-string is not subtype of native type 'Ray\\MediaQuery…'.


(varTag.nativeType)